src/stories/Blocks/reservation/reservation.scss

```diff
@@ -1,8 +1,18 @@
 .reservation-modal {
-  background-color: #f6f5f0;
+  background-color: $c-global-primary;
   width: 100%;
 }
 
+.reservation-modal--confirm {
+  display: grid;
+  place-content: center;
+
+  button {
+    width: 193px;
+    margin: 0 auto;
+  }
```

Comment:
Consider if it is possible to target a CSS class instead of a HTML tag.

Examples: 

- Add an `reservation-modal__button` class to the button which sets the width.
- Add a modifier to the button element e.g. `btn--wide` which sets the width.

Is `margin: 0 auto;` necessary when you use `place-content: center;`?